node_modules/gh-admins/tests/test-admins.js

```diff
@@ -24,87 +24,280 @@ var TestsUtil = require('gh-tests/lib/util');
 
 var AdminsTestsUtil = require('./util');
 
+var anonymousGlobalAdminClient = null;
+var globalAdminClient = null;
+
+beforeEach(function(callback) {
```

Comment:
This should go in the describe block, as this will now get executed for every single test in the application